graph/telemetry/istio/appender/unused_node.go

```diff
@@ -66,11 +66,11 @@ func (a UnusedNodeAppender) buildUnusedTrafficMap(trafficMap graph.TrafficMap, n
 	unusedTrafficMap := graph.NewTrafficMap()
 
 	for _, s := range services {
-		id, nodeType := graph.Id(namespace, s.Service.Name, "", "", "", "", a.GraphType)
+		id, nodeType := graph.Id(graph.Unknown, namespace, s.Service.Name, "", "", "", "", a.GraphType)
```

Comment:
Perhaps this could be the "home" cluster?

Reply:
Hmm... Definitely, it's something to consider for another PR. This is the result when enabling unused nodes:

![image](https://user-images.githubusercontent.com/23639005/101555217-cdb7c480-397d-11eb-8573-9e77eed5c482.png)

I see repeated nodes...

Reply:
OK, something to handle.  I did not try Unused Nodes option.

Reply:
added as subtask to https://github.com/kiali/kiali/issues/1833